nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/RelationTagService.java

```diff
@@ -212,7 +212,7 @@ public boolean canUntag(CoreSession session, String docId, String label) {
         return canUntag;
     }
 
-    protected static class UnrestrictedCanRemoveTagging extends UnrestrictedSessionRunner {
```

Comment:
This cleanup seems unrelated to the ticket?

Reply:
The class using the `cleanLabel` method, that is using the non static method `isTagSanitizationEnabled`